x-pack/elastic-agent/pkg/agent/warn/warn.go

```diff
@@ -21,4 +21,5 @@ func LogNotGA(log *logger.Logger) {
 // PrintNotGA writes to the received writer that the Agent is not GA.
 func PrintNotGA(output io.Writer) {
 	fmt.Fprintln(output, message)
+	fmt.Fprintln(output)
```

Comment:
This removes the print of the message?

Reply:
No it just adds an extra blank line between the warning and the next message. It just makes it easier to read when using the sub-commands.

Reply:
Ah I've missread the diff :(